### DISCUSSION TITLE
Renaming the Tool Item

### DISCUSSION THREAD

**Discussion by HappyCrafter156** [maruohon/litematica#323] [Q&A] [answered]
So I just got litematica a few months ago, and was wondering if you could make your tool item a renamed one. In the configs menu, I already tried to do this:

minecraft:stick{display:{Name:'{"Example Name"}'}}

Do I have to do it some other way or is there no way to make it a named item?

**Comment by maruohon** [comment, marked as answer]
This currently only works in the main development version, ie. 1.12.2 LiteLoader. The 1.13.2+ versions still use older code for the tool item setting (among countless other things), and they don't yet support the improved tool item handling.

In general I would recommend using whatever item you always have on you anyway (such as your food item) as the tool, and then just toggle the tool functionality on/off as needed (by default with `M + T`).

**Reply by HappyCrafter156** [reply]
Okay, thanks for an answer!
